### PR TITLE
feat: adding git amend command

### PR DESCRIPTION
**Pull request by iloveitaly**

### CODE REVIEW

**Comment by iloveitaly** [comment]
@bigH this has been working for me the last couple days!

**Comment by bigH** [comment]
There's only one issue. `amend` is misspelled `ammend` in this PR. Once fixed, I'm happy to merge this :)

File: lib/modules/status.sh [hunk 1]
```diff
@@ -13,7 +14,7 @@ Type to filter. '"${WHITE}Enter${NORMAL} to ${GREEN}ACCEPT${NORMAL}"'
 
   '"${GRAY}-- (${NORMAL}*${GRAY}) editor: ${MAGENTA}${EDITOR} ${NORMAL}${GF_EDITOR_ARGS}${NORMAL}"'
   '""'
-  '"               ${GREEN}stage -p ${BOLD}⇡  ${NORMAL}${WHITE}${GIT_FUZZY_STATUS_ADD_PATCH_KEY}${NORMAL}       * ${GREEN}${BOLD}edit ✎${NORMAL}  ${WHITE}$GIT_FUZZY_STATUS_EDIT_KEY${NORMAL}"'
+  '"${GREEN}ammend  ${NORMAL}${WHITE}${GIT_FUZZY_STATUS_AMEND_KEY}${NORMAL}  ${GREEN}stage -p ${BOLD}⇡  ${NORMAL}${WHITE}${GIT_FUZZY_STATUS_ADD_PATCH_KEY}${NORMAL}       * ${GREEN}${BOLD}edit ✎${NORMAL}  ${WHITE}$GIT_FUZZY_STATUS_EDIT_KEY${NORMAL}"'
```
Comment (iloveitaly) [inline lib/modules/status.sh]:
```suggestion
  '"${GREEN}amend  ${NORMAL}${WHITE}${GIT_FUZZY_STATUS_AMEND_KEY}${NORMAL}  ${GREEN}stage -p ${BOLD}⇡  ${NORMAL}${WHITE}${GIT_FUZZY_STATUS_ADD_PATCH_KEY}${NORMAL}       * ${GREEN}${BOLD}edit ✎${NORMAL}  ${WHITE}$GIT_FUZZY_STATUS_EDIT_KEY${NORMAL}"'
```

**Comment by iloveitaly** [comment]
lol, whoops!

**Pull request merged**